test/fixtures.js

```diff
@@ -390,7 +390,7 @@ return [
                     "created_at": "2014-08-05T11:31:50.908974",
                     "id": 3,
                     "lang": "en",
-                    "messageset_id": 1,
+                    "messageset_id": 2,
```

Comment:
I don't think anything in the current code as you have it now can set `messageset_id` to `2` (since you hard coded it to `3`). That would suggest that these fixtures are unused? If so, we'll need to add a test for the refugee subscription case so that that case is tested too.